<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainmenu</class>
 <widget class="QDialog" name="mainmenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1277</width>
    <height>917</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">#mainmenu{ 
	border-image: url(:/images/Myresources/background.png);
}</string>
  </property>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>40</y>
     <width>871</width>
     <height>811</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>643</width>
     <height>577</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">#frame{
	border: 2px;
    border-radius: 10px;
	background-color: rgba(30, 30, 30, 150);
}</string>
   </property>
   <widget class="QListWidget" name="listWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>390</y>
      <width>191</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>161</width>
      <height>111</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="page"/>
    <widget class="QWidget" name="page_2"/>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>-140</x>
      <y>690</y>
      <width>482</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_7">
     <item alignment="Qt::AlignmentFlag::AlignHCenter">
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Berlin Sans FB</family>
         <pointsize>16</pointsize>
         <bold>false</bold>
        </font>
       </property>
       <property name="text">
        <string>Patient's Dashboard</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="name">
         <property name="font">
          <font>
           <family>Berlin Sans FB</family>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout">
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QLabel" name="age">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string>Age</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="agedata">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item alignment="Qt::AlignmentFlag::AlignHCenter">
            <widget class="QLabel" name="label_4">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string>Blood Group</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="label_8">
               <property name="font">
                <font>
                 <family>Berlin Sans FB</family>
                </font>
               </property>
               <property name="text">
                <string>Height</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox">
               <property name="font">
                <font>
                 <family>Berlin Sans FB</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#comboBox{
background-color: transparent;
    border: none;
color: rgb(170, 255, 255);         /* Change text color if needed */
}

comboBox::drop-down {
    background: transparent;
    border: none;
}

comboBox QAbstractItemView {
    background: rgba(0, 0, 0, 100);   /* popup list transparency */
    color: white;
}</string>
               </property>
               <item>
                <property name="text">
                 <string>(Ft)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>(Cm)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_9">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label_6">
               <property name="font">
                <font>
                 <family>Berlin Sans FB</family>
                </font>
               </property>
               <property name="text">
                <string>Weight</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QComboBox" name="comboBox_2">
               <property name="font">
                <font>
                 <family>Berlin Sans FB</family>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">#comboBox_2{
background-color: transparent;
    border: none;
color: rgb(170, 255, 255);         /* Change text color if needed */
}

comboBox_2::drop-down {
    background: transparent;
    border: none;
}

comboBox_2 QAbstractItemView {
    background: rgba(0, 0, 0, 100);   /* popup list transparency */
    color: white;
}</string>
               </property>
               <item>
                <property name="text">
                 <string>(Lb)</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>(Kg)</string>
                </property>
               </item>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Berlin Sans FB</family>
              </font>
             </property>
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QListView" name="listView">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>520</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="btnmedicine">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>201</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>add medicine</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>100</y>
      <width>256</width>
      <height>192</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>211</width>
      <height>192</height>
     </size>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#tablewidget{
	border: 2px;
    border-radius: 10px;
	background-color: rgba(30, 30, 30, 150);
}

</string>
    </property>
    <column>
     <property name="text">
      <string>doctor name</string>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
      </font>
     </property>
    </column>
    <column>
     <property name="text">
      <string>appointment date</string>
     </property>
     <property name="font">
      <font>
       <family>Berlin Sans FB</family>
      </font>
     </property>
    </column>
   </widget>
   <widget class="QWidget" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>446</x>
      <y>298</y>
      <width>16</width>
      <height>194</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>520</y>
      <width>127</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#pushButton{
color :rgb(0, 255, 255);
background-color: transparent;
  border: none;
}</string>
    </property>
    <property name="text">
     <string>show graph analytics</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>300</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Berlin Sans FB</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>caretakers</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
